ocs/ocp.py

```diff
@@ -146,8 +146,10 @@ def delete(self, yaml_file=None, resource_name='', wait=True):
             command += f"{self.kind} {resource_name}"
         else:
             command += f"-f {yaml_file}"
-        if wait:
-            command += " --wait=true"
+
+        # default value for wait is true
```

Comment:
Right. Added it because it was my assumption before I discovered the real cause - https://bugzilla.redhat.com/show_bug.cgi?id=1715627
And forgot to change it back. Will revert

Reply:
@RazTamir actually, it seems that the default is true..I see this when trying to delete a PVC while it is attached to a pod and its status is Terminating (the pod has to be deleted). The deletion never finishes and gets stuck.